tests_config/run_spec.js

```diff
@@ -16,6 +16,36 @@ const prettier = !TEST_STANDALONE
   ? require("prettier/local")
   : require("prettier/standalone");
 
+// TODO: make these test files support `babel-ts` parser
```

Comment:
These tests aren't TODO. They all contain invalid code. TS supports them because its parser is super permissive by design (as it powers the language service, which should work with invalid/incomplete code in IDEs), but it's unlikely Babel will ever parse them.

Reply:
Good to know.

Reply:
No, we shouldn't. But the `toThrow` assertion still is a good thing to have.

Reply:
Okay, I'll keep the `disableBabelTS` and add `toThrow`